packages/jsii-pacmak/lib/targets/go/package.ts

```diff
@@ -266,23 +263,9 @@ export abstract class Package {
       return;
     }
 
-    emit.call(
-      this,
-      join(
-        this.directory,
-        `${this.packageName}_${type.name}__runtime_type_checks.go`,
-      ),
-      false,
-    );
+    emit.call(this, join(this.directory, `${type.name}__checks.go`), false);
```

Comment:
Does this not risk collisions if two namespaces have the same leaf type name?

Reply:
No, they are in nested folders